python_modules/dagster-graphql/dagster_graphql/schema/roots/query.py

```diff
@@ -244,6 +246,12 @@ class Meta:
         description="Retrieve a job by its location name, repository name, and job name.",
     )
 
+    resources = graphene.Field(
+        non_null_list(GrapheneResource),
```

Comment:
should this be nullable so the exception on bad input just returns `resources: null` instead of whatever it does if this propagates up ?